src/wp-includes/functions.php

```diff
@@ -7637,12 +7637,40 @@ function wp_unique_id( $prefix = '' ) {
 function wp_cache_get_last_changed( $group ) {
 	$last_changed = wp_cache_get( 'last_changed', $group );
 
-	if ( ! $last_changed ) {
-		$last_changed = microtime();
-		wp_cache_set( 'last_changed', $last_changed, $group );
+	if ( $last_changed ) {
+		return $last_changed;
 	}
 
-	return $last_changed;
+	return wp_cache_set_last_changed( $group );
+}
+
+/**
+ * Sets last changed date for the specified cache group to now.
+ *
+ * @since n.e.x.t.
```

Comment:
```suggestion
 * @since 6.3.0
```
WordPress does not follow `n.e.x.t` format but instead uses the next major version. If needed, update it to set a milestone.

```suggestion
 * @since n.e.x.t
```
Or remove last period from `n.e.x.t`